tests/indent/c/unfinished_comment.c

```diff
@@ -0,0 +1,2 @@
+/*
+ *
```

Comment:
This could be fixed by
```
((comment) @aligned_indent
 (#lua-match? @aligned_indent "^/\*"))
```
but strangly this breaks the unfinished call expression in `tests/indent/c/aligned_indent.c`